poetry/factory.py

```diff
@@ -81,6 +81,16 @@ def create_poetry(
         if "readme" in local_config:
             package.readme = Path(poetry_file.parent) / local_config["readme"]
 
+        # Add any manually defined or automatically discovered license files.
+        if "license-files" in local_config:
+            for license_file in local_config["license-files"]:
+                package.license_files.add(Path(poetry_file.parent) / license_file)
+        else:
+            for license_file in poetry_file.parent.glob("LICENSE*"):
```

Comment:
i'm not sure whether the stdlib's globbing implementation supports this, but `{COPYING*,LICENSE*}` should be enough.

Reply:
The standard library globbing doesnt support multiple patterns unfortunately. https://docs.python.org/3/library/glob.html#glob.glob

I've just tried a few variations but none of them are particularly nice looking, and `black` has a tendency to reformat them over more lines than the code they would replace.

```python
            for glob_pattern in ("LICENSE*", "COPYING*"):
                for license_file in poetry_file.parent.glob(glob_pattern):
                    package.license_files.add(
                        license_file.relative_to(poetry_file.parent)
                    )
```
The code above was the nicest result and I'm happy to change it but it is at the end of the day... really just a fancier way of doing the same thing as the original code below.
```python
            for license_file in poetry_file.parent.glob("LICENSE*"):
                package.license_files.add(license_file.relative_to(poetry_file.parent))
            for license_file in poetry_file.parent.glob("COPYING*"):
                package.license_files.add(license_file.relative_to(poetry_file.parent))
```

Reply:
@funkyfuture Any more thoughts on this?